cpack/debian/prerm

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+# Automatically added by dh_installinit/12.10
```

Comment:
And here.

Reply:
I removed Comments of dbhelper commands on all scripts